xchange-okcoin/src/main/java/org/knowm/xchange/okcoin/service/OkCoinFuturesTradeService.java

```diff
@@ -277,13 +277,13 @@ public OpenOrdersParams createOpenOrdersParams() {
       OkCoinFuturesOrderQueryParams myParams = (OkCoinFuturesOrderQueryParams) orderQueryParam;
       CurrencyPair currencyPair = myParams.getCurrencyPair();
       FuturesContract reqFuturesContract = myParams.futuresContract;
-      long orderId = myParams.getOrderId() != null ? Long.valueOf(myParams.getOrderId()) : -1;
+      long orderId = myParams.getOrderId() != null ? Long.parseLong(myParams.getOrderId()) : -1;
 
       if (ordersToQuery.get(currencyPair) == null) {
         Set<String> orderSet = new HashSet<>();
```

Comment:
Collections.singleton("a"); lets use this for the single message hashsets and lets just do it inline of the put method

Reply:
Done